README.md

```diff
@@ -1 +1,180 @@
+  </ul>
+</details>
+
+<details>
+  <summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- User authentication
+- Users can create and view transaction categories
+- Users can create and view transactions
+- Users can monitor total spend on each category
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link]() IF YOU ARE REVIEWING THIS I CANT DEPLOY MY WORK CAUSE I DONT KNOW HOW TO COULD YOU PLEASE HELP ME WITH INSTRUCTONS OR PLAYBY PLAY? THANKS!!- [Video Documentation](https://www.loom.com/share/727e92f38770461c9faaaecf61df961a?sid=95f9b826-8cdf-4e1b-92fb-fe023f13860f)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+```sh
+ git
+ bundle
+ Your favorite text editor
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```
+https://github.com/Booze33/budget-app.git
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd budget-app
+  bundle install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  rails server - Starts the development server
+  rails console - Starts the interactive command line
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  rspec - Runs the test suite
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+ 👤 **Iheagwara Augustine**
+
+- GitHub: [@stino-x](https://github.com/stino-x)
+
+  
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] Add a feature to allow users to follow other users
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please leave a star🤩
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- [Microverse](https://www.microverse.org/)
+- Original Design Idea by [Gregoire Vella on Behance](https://www.behance.net/gregoirevella)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] Please correct the license link as this is broken to be seem more professional.

Reply:
hi!! i think it works just hold ctrl+click it takesyou to the licencse thanks!!